Makefile

```diff
@@ -25,6 +25,8 @@ circle-setup:
 	cp -R ./* $(DRUPAL_SITE_PATH)/profiles/ding2/
 	# Install the site using the ding2 profile
 	cd $(DRUPAL_SITE_PATH) && drush site-install ding2 --db-url=mysql://ubuntu@127.0.0.1/circle_test -y ding2_module_selection_form.providers_selection=connie
+	# Apply Drupal core patches
+	cd $(DRUPAL_SITE_PATH) && git apply -v $(DRUPAL_SITE_PATH)/profiles/ding2/patches/drupal_core.robots.txt.ding2.patch
```

Comment:
Why not include it in `drupal.make` instead [like we do for other Core patches](https://github.com/ding2/ding2/blob/master/drupal.make#L10)?